docs/fleet/_index.md

```diff
@@ -18,103 +18,62 @@ aliases:
 menuindent: true
 ---
 
-The [Viam app](https://app.viam.com) provides fleet management allowing you to work on any number of machines alone or in collaboration with others.
-You can manage and control your fleet of machines from the Viam app, using the [CLI](/fleet/cli/), or using the [cloud API](/build/program/apis/cloud/).
+Viam fleet management allows you to organize, manage, and control any number of machines alone or in collaboration with others.
+You can manage and control your fleet of {{< glossary_tooltip term_id="machine" text="smart machines" >}} from the Viam app, using the [CLI](/fleet/cli/), or using the [cloud API](/build/program/apis/cloud/).
 
 ## Work with groups of machines
 
-With Viam, you can organize {{< glossary_tooltip term_id="machine" text="smart machines" >}} into {{< glossary_tooltip term_id="location" text="locations" >}} and {{< glossary_tooltip term_id="organization" text="organizations" >}}.
+To organize your fleet you use
 
-For example, you may have separate organizations for your machines at home and at work:
+- {{< glossary_tooltip term_id="organization" text="organizations" >}}: the highest level grouping, generally used for different companies.
+- {{< glossary_tooltip term_id="location" text="locations" >}}: virtual groupings of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
 
-<!-- this is a very small gif - conversion to mp4 caused issues -->
-<img src="/manage/organizations.gif" alt="An organization for personal machines and one for work machines.">
+Each machine resides in a location.
 
-Inside an organization, you can organize machines into one or more locations:
+<!-- TODO: Add topology based on Jon's draft -->
 
 ![An image of two locations, New York, and Chicago, in one organization, Good Robots](/fleet/locations.png)
 
-If you are managing a fleet, you can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} when [configuring your machines](/build/configure/), allowing you to use the same configuration for multiple machines.
+The organization structure enables you to:
 
-## Use Viam for collaboration
+- [configure groups of machines](/build/configure/) with reusable {{< glossary_tooltip term_id="fragment" text="fragments" >}} that configure, among other things:
```

Comment:
I feel like this makes me think i have to use fragments to do the list below

Reply:
ok simplifying and removing list - I think that's a good call